core/domain/wipeout_service_test.py

```diff
@@ -511,6 +511,8 @@ def test_pre_delete_user_exp_user_with_voice_artist_role_is_deassigned(
             self.user_2_id,
             feconf.ROLE_VOICE_ARTIST
         )
+        rights_manager.unpublish_exploration(
```

Comment:
Why is this added (and should we have another test like the previous one with this statement removed)?

Reply:
Added a second test, one is for private and one for public exploration.